koiki/woocommerce/models.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+
+class Vendor():
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+    def __eq__(self, other):
+        if not isinstance(other, Vendor):
+            return NotImplemented
+
+        return self.id == other.id and self.name == other.name
+
+
+class LineItem():
+    def __init__(self, line_item):
+        self.quantity = line_item['quantity']
+        self.metadata = line_item['meta_data']
+        vendor = self._find_vendor()
+        self.vendor = Vendor(vendor[0], vendor[1])
```

Comment:
Maybe using Vendor(id=vendor[0], name=vendor[1]) for more code clarity. And even return directly the Vendor object from within the _find_vendor method ?

Reply:
I did that but then I thought I prefered to keep the method with a single responsibility. Yes, I know it may be a micro-optimization 😂